app/views/item_seasons/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<p id="notice"><%= notice %></p>
+
+<h1 class='page__title'> <%= I18n.t('item_season.index.title') %> </h1>
+
+<table class='produce-table'>
+  <% I18n.t('date.month_names').each do |month_name|  %>
```

Comment:
Hm, I feel like logic here is somehow inverted. We are getting an array of months from translations to then translate back to an index. This also forces you to handle the fact that `month_names` has `nil` as the first item.

I'd prefer to get the indices from a model to then derive a name from it. For example:
```erb
<% ItemSeason::MONTHS.each do |month_index| %>
  <%= I18n.t('date.month_names')[month_index] %>
<%  end %>
```

WDYT?

Reply:
I completely agree, I had doubts myself, and thought it was a rather roundabout way to go at it. This makes much more sense.

The only thing I'm suggesting is using our `formatted_month_name` helper instead of `I18n.t('date.month_names')[month_index]`. I think it's more readable, and it's already available to us :]

Reply:
Makes sense. Then we should move the helper to `application_helpers`.